Tests/Asm/Ansible/Command/AnsiblePlaybookTest.php

```diff
@@ -824,12 +818,6 @@ public function testExtraVars(): void
                 'input' => ['key1' => 'value1', 'key2' => 'value2'],
                 'expect' => '--extra-vars=key1=value1 key2=value2',
             ],
-            // Test valid JSON.
-            [
-                'input' => '{ "key1": "value1", "key2": "value2" }',
```

Comment:
@maschmann @bambamboole I think that the json or empty string vars tests have been removed?